common/src/main/scala/com/gu/monitoring/MemberDataAPIMetrics.scala

```diff
@@ -0,0 +1,13 @@
+package com.gu.monitoring
+
+import Metrics._
+class MembersDataAPIMetrics(val name: String) extends CloudWatch(Seq(
+  productName(name)
+)) {
+
+  def putMembersDataAPIUpdated(): Unit = put(s"$name-members-data-api-updated")
```

Comment:
This shouldn't return `Unit`, it should return a `Future` which completes when the `put` request has completed.